src/Analyzer.BicepProcessor/Analyzer.BicepProcessor.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you enforce summary comments on public functions. Here is an example of how to enable that in the csproj https://github.com/Azure/template-analyzer/blob/c9f38aa454ced1d6d689bee9d9c37fdfe2ada826/src/Analyzer.JsonRuleEngine/Analyzer.JsonRuleEngine.csproj#L10

https://github.com/Azure/template-analyzer/blob/c9f38aa454ced1d6d689bee9d9c37fdfe2ada826/src/Analyzer.JsonRuleEngine/Analyzer.JsonRuleEngine.csproj#L11

Reply:
@norbusonam There's a PR that generalizes this now https://github.com/Azure/template-analyzer/pull/151 You can remove the additional lines in the csproj.